rosidl_runtime_c/src/string_functions.c

```diff
@@ -131,6 +131,18 @@ rosidl_runtime_c__String__assign(
     str, value, strlen(value));
 }
 
+bool
+rosidl_runtime_c__String__copy(
+  const rosidl_runtime_c__String * input,
+  rosidl_runtime_c__String * output)
+{
+  if (!input) {
```

Comment:
do we need to check `!output` here?

Reply:
No need to, `rosidl_runtime_c__String__assignn()` will do that check.